specs/language/declarations.tex

```diff
@@ -96,10 +96,15 @@
 assignment-expression in an initializer-list and performing a depth-first
 traversal accessing each subobject of the assignment-expression.
 
-\p An initializer-list is a valid initializer if for each element \(E_n\) in the
-target object's flattened ordering there is a corresponding initializer \(I_n\)
-in the flattened initializer sequence which can be implicitly converted to the
-element's type.
+\p If the target object is an array of unknown size, the object is assumed to
+have \(m\) possible elements during parsing, where \(m>0\).
+
+\p An initializer-list is a valid initializer if for each element
+\(E_{n \bmod m}\) in the target object's flattened ordering there is a
+corresponding initializer \(I_n\) in the flattened initializer sequence, which
```

Comment:
The word initializer here doesn't refer to the concept of initializer in the grammar right? It refers to an initializer-clause? In a flattened initializer sequence, I_n would be something like '5', not something like {1,2,3}?